build/generate-docs.ts

```diff
@@ -180,6 +181,8 @@ function typeToMarkdownLink(type: string): string {
         case 'paint':
         case 'layout':
             return ` [${type}](layers.md#${type.toLocaleLowerCase()})`;
+        case 'projectionconfig':
+            return ' [projectionConfig](projection.md)';    
```

Comment:
Is this really needed here?
This is used for the docs to allow link to an object type related to the expressions I think, and I'm not sure projectionConfig can be a return type of an expression, right?

Reply:
This is an overwrite so that the url of the main Projection page doesn't get a /projectionConfig url.

<img width="686" alt="Screenshot 2024-11-13 at 09 04 55" src="https://github.com/user-attachments/assets/91db9f21-f77c-412b-b36b-ec249ec3cc4d">

If the line is removed this pops up
`WARNING -  Doc file 'root.md' contains a link 'projectionConfig.md', but the target is not found among documentation files.`

Reply:
Ah, I see...
I'm trying to think if there's a way to solve this elegantly, but I can't think of a good idea at the moment.

Reply:
I'm not sure this is a good idea, but in proj4js they use the term `ProjectionDefinition` for the values related to the projection:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/proj4/index.d.ts

I'm not sure what I think about it though - this is for the "internal" type, the one used in the expressions, not the root projection type.